packages/graphql_flutter/example/graphql_chat/lib/view/home/home_body.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:graphql_chat/api/query.dart';
+import 'package:graphql_chat/model/chat.dart';
+import 'package:graphql_flutter/graphql_flutter.dart';
+import 'package:logger/logger.dart';
+
+class HomeViewBody extends StatelessWidget {
```

Comment:
Should we make sure our example also has mutations?

Reply:
Yes, this required some work to do also in the API but we could integrate it